tests/test_tuya_thermostat.py

```diff
@@ -0,0 +1,85 @@
+"""Tests for Tuya Smoke Detector."""
+
+import pytest
+from zigpy.zcl import foundation
+from zigpy.zcl.clusters.hvac import Thermostat
+
+from tests.common import ClusterListener
+import zhaquirks
+from zhaquirks.tuya.mcu import TuyaMCUCluster
+
+zhaquirks.setup()
+
+
+@pytest.mark.parametrize(
+    "model,manuf,msg,attr,value",
+    [
+        (
+            "_TZE204_p3lqqy2r",
+            "TS0601",
```

Comment:
I feel like we could also hard-code model and manufacturer directly in the test.
If another thermostat is added, it might be worth just copying the entire (small) test. The`parametrize` entries will become very long otherwise.